presto-native-execution/CMakeLists.txt

```diff
@@ -177,8 +187,9 @@ set(OPENSSL_ROOT_DIR "${OPT_OPENSSL_DIR}/opt/openssl@1.1")
 set_source(OpenSSL)
 resolve_dependency(OpenSSL REQUIRED)
 
-# TODO: cannot be found via find_package/resolve_dependency
-find_library(LIBSODIUM_LIBRARY NAMES sodium)
+# Note: the uppercase name here is correct.
+set_source(Sodium)
+resolve_dependency(Sodium)
```

Comment:
This didn't work before because the name defined in the Find module uses uppercase (bundled with velox).